spec/lib/hyrax/analytics_spec.rb

```diff
@@ -2,11 +2,12 @@
 RSpec.describe Hyrax::Analytics do
   before do
     ENV['GOOGLE_ANALYTICS_ID'] = 'UA-XXXXXXXX'
-    ENV['GOOGLE_OAUTH_APP_NAME'] = "My App Name"
-    ENV['GOOGLE_OAUTH_APP_VERSION'] = "0.0.1"
-    ENV['GOOGLE_OAUTH_PRIVATE_KEY_PATH'] = "/tmp/privkey.p12"
-    ENV['GOOGLE_OAUTH_PRIVATE_KEY_SECRET'] = "s00pers3kr1t"
-    ENV['GOOGLE_OAUTH_CLIENT_EMAIL'] = "oauth@example.org"
+    ENV['GOOGLE_OAUTH_APP_NAME'] = 'My App Name'
+    ENV['GOOGLE_OAUTH_APP_VERSION'] = '0.0.1'
+    ENV['GOOGLE_OAUTH_PRIVATE_KEY_PATH'] = '/tmp/privkey.p12'
+    ENV['GOOGLE_OAUTH_PRIVATE_KEY_VALUE'] = ''
```

Comment:
added this line. the rest of the env changes are unifying them with single quotes.